app/renderer/js/main.js

```diff
@@ -608,7 +608,7 @@ class ServerManagerView {
 							message: 'Are you sure you want to disconnect this organization?'
 						}, response => {
 							if (response === 0) {
-								if (EnterpriseUtil.isAdminOnly('presetOrganizations')) {
+								if (EnterpriseUtil.isAdminOnly('presetOrganizations') && this.presetOrgs.includes(DomainUtil.getDomain(index).url)) {
```

Comment:
This commit probably needs to be squash with the commit that introduced the feature.